force-app/main/default/classes/SOQL.cls

```diff
@@ -161,47 +279,76 @@ public with sharing class SOQL {
      * @param stage The Stage value to filter Opportunities.
      * @return The total revenue.
      */
-    public static Decimal getTotalRevenueByStage(String stage) {
+    public Decimal getTotalRevenueByStage(String stage) {
         // Use AggregateQuery to get the total revenue
+        AggregateResult[] groupedResults = [
+            SELECT SUM(Amount) totalRevenue
+            FROM Opportunity
+            WHERE StageName = :stage OR Amount > 10000
+        ];
+
+        // Retrieve the total revenue from the query result
+        Decimal totalRevenue = (Decimal)groupedResults[0].get('totalRevenue');
 
         // Return the total revenue
-        return null; // Replace null with the variable you used to store the result
+        return totalRevenue != null ? totalRevenue : 0; // Replace null with the variable you used to store the result
     }
 
     /**
      * Question 13
      * Retrieves the first 10 Accounts ordered by Name.
      * @return A list of the first 10 Accounts ordered by Name.
      */
-    public static List<Account> getFirstTenAccounts() {
-        return null; // Replace null with the variable you used to store the result
+    public List<Account> getFirstTenAccounts() {
+        List<Account> accounts = [SELECT Id, Name FROM Account ORDER BY Name LIMIT 10];
+        return accounts; // Replace null with the variable you used to store the result
     }
     
     /**
      * Question 14
      * Retrieves the Opportunities with an Amount greater than 100,000, ordered by CloseDate, and limited to 5 records.
      * @return A list of Opportunities with an Amount greater than 100,000, ordered by CloseDate, and limited to 5 records.
      */
-    public static List<Opportunity> getHighValueOpportunities() {
-        return null; // Replace null with the variable you used to store the result
+    public List<Opportunity> getHighValueOpportunities() {
+        List<Opportunity> highValueOpportunities = [
+        SELECT Id, Name, Amount, CloseDate 
+        FROM Opportunity 
+        WHERE Amount > 100000 
+        ORDER BY CloseDate 
+        LIMIT 5
```

Comment:
Try creating variables for the number in this so that they are easier to understand.